scripts/exgdas_global_marine_analysis_vrfy.py

```diff
@@ -100,12 +101,18 @@ def plot_zonal_slice(config):
 
 
 comout = os.getenv('COMOUT')
+data = os.getenv('DATA')
 cyc = os.getenv('cyc')
 bcyc = str((int(cyc) - 3) % 24)
 
-# TODO: do not write to COM, instead dump figures in DATA and copy to COM.
 grid_file = os.path.join(comout, 'gdas.t'+bcyc+'z.ocngrid.nc')
 
+# for eva
+diagdir = os.path.join(comout, 'diags')
+project_source_dir = os.getenv('PROJECT_SOURCE_DIR')
```

Comment:
this is not going to be available if you are not in CMake land...

Reply:
this would have been superfluous anyway - turns out I'd caught it already